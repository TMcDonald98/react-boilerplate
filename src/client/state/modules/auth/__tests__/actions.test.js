import {
  setApiKey
} from '../actions';

describe('auth/actions', () => {
  describe('setApiKey', () => {
    it('should set the api key', () => {
      const testApiKey = 'test';
      const result = setApiKey(testApiKey);

      expect(result).toMatchSnapshot('setApiKey');
    });
  });
});
