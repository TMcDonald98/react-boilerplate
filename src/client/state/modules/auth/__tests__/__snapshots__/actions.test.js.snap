// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`auth/actions setApiKey should set the api key: setApiKey 1`] = `
Object {
  "payload": "test",
  "type": "SET_API_KEY",
}
`;
